beast/examples/phat_small/run_beast.py

```diff
@@ -188,7 +188,8 @@
            datamodel.noisefile,
            datamodel.astfile,
            modelsedgrid,
-           absflux_a_matrix=datamodel.absflux_a_matrix)
+           absflux_a_matrix=datamodel.absflux_a_matrix,
+           use_rate=False)
```

Comment:
Is this `False` because the PHAT photometry is only magnitudes?

Reply:
Yep.  The phat_small and regression tests are based on *old* ASTs.  So, only output magnitudes available (no rate columns).